common/build/eslint-plugin-fluid/src/test/no-hyphen-after-jsdoc-tag.test.js

```diff
@@ -38,21 +38,30 @@ describe(`Do not allow \`-\` following JSDoc/TSDoc tags (eslint ${eslintVersion}
 		const error1 = result.messages[0];
 		assert.strictEqual(error1.message, expectedErrorMessage);
 		assert.strictEqual(error1.line, 8);
-		assert.strictEqual(error1.column, 4); // 1-based, inclusive
-		assert.strictEqual(error1.endColumn, 13); // 1-based, exclusive
+		assert.strictEqual(error1.column, 12); // 1-based, inclusive
```

Comment:
Note for reviewers: Indexes now correspond with the hyphen characters (and surrounding whitespace), rather than the TSDoc tag, so all of the ranges have shifted forward (and shrunk).